cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -79,15 +79,21 @@ def _make_glob_from_args(
     """
 
     if reporter and volumes and page:
-        reporter_key = f"law.free.cap.{reporter}.{volumes[0]}"
-        glob_path = f"{settings.MEDIA_ROOT}/harvard_corpus/{reporter_key}/{page}.*.json"
+        glob_path = os.path.join(
+            settings.MEDIA_ROOT,
+            "harvard_corpus",
+            f"law.free.cap.{reporter}.{volumes[0]}",
+            f"{page}.*.json",
+        )
         return [glob_path]
     elif reporter and volumes:
         glob_paths = []
         for volume in volumes:
-            reporter_key = f"law.free.cap.{reporter}.{volume}"
-            glob_path = (
-                f"{settings.MEDIA_ROOT}/harvard_corpus/{reporter_key}/*.json"
+            glob_path = os.path.join(
+                settings.MEDIA_ROOT,
+                "harvard_corpus",
+                f"law.free.cap.{reporter}.{volume}",
+                ".*.json",
```

Comment:
Is the leading dot on this line correct? 
```suggestion
                "*.json",
```